poly/joint_test.go

```diff
@@ -0,0 +1,99 @@
+package poly
+
+import (
+	"fmt"
+	"testing"
+)
+
+/////// TESTING ///////
+
+func TestReceiverAddDealer(t *testing.T) {
```

Comment:
Oops, yes - I should have put this note on a different function, e.g., TestRightDealer... below. :)